test/powershell/Modules/Microsoft.PowerShell.Utility/New-Object.Tests.ps1

```diff
@@ -1,203 +1,211 @@
 Describe "New-Object" -Tags "CI" {
+    It "Support 'ComObject' parameter on platforms" {
+        if ($IsLinux -or $IsMacOs ) {
+            { New-Object -ComObject "Shell.Application" } | ShouldBeErrorId "NamedParameterNotFound,Microsoft.PowerShell.Commands.NewObjectCommand"
+        } else {
+            { New-Object -ComObject "Shell.Application" } | Should Not Throw
```

Comment:
This will fail when running tests on NanoServer ... Maybe change it to check if `New-Object` has the parameter `ComObject` instead of creating the `Shell.Application` object?
```
$s = gcm New-Object
$s.Parameters.ContainsKey("ComObject")
True
```